tests/unit/test_args.py

```diff
@@ -907,3 +945,15 @@ def test_sdk_endpoint_legacy_set_without_builtin_data(self):
             legacy_endpoint_url='https://my.legacy.endpoint.com',
         )
         self.assertEqual(bins['SDK::Endpoint'], None)
+
+
+class TestProtocolPriorityList:
```

Comment:
We should provide an error message here to make it more obvious when debugging this error.